src/modules/auth/controller.js

```diff
@@ -28,6 +31,34 @@ const tryAuth = async (req, res, next) => {
   }
 };
 
+const refreshToken = async (req, res, next) => {
+  try {
+    const user = userService.getUser({ id: req.user.id });
+
+    if (!user) {
+      next(new UnauthorizedError(INVALID_TOKEN_MESSAGE));
+    }
+
+    const tokenPair = authService.generateTokenPair({
+      id: req.user.id,
+      email: req.user.email,
+    });
+
+    const bearerToken = authService.processTokenPair(res, tokenPair);
+
+    return res.sendResponse(StatusCodes.OK, { accessToken: bearerToken });
+  } catch (err) {
+    return next(err);
+  }
+};
+
+const logout = (req, res, next) => {
+  res.clearCookie('refreshToken');
+  return res.sendResponse(StatusCodes.OK);
+};
+
 router.post('/', validator(validateAuth), tryAuth);
+router.put('/', authMiddleware.refresh, refreshToken);
+router.delete('/', logout);
```

Comment:
Based on the ticket https://trello.com/c/KZdySDwv
Log out - is an action for authorized users
So where is the verification logic?